mflowgen/common/cadence-xcelium-sim/run_sim.sh

```diff
@@ -66,6 +73,14 @@ for f in inputs/*.sv; do
   ARGS="$ARGS $f"
 done
 
+if [ ${use_sdf} = "True" ]; then
+  # Annotate the subcomponents first and then do top
+  if [ -f "sub_cmd.cmd" ]; then
+    ARGS="$ARGS -sdf_cmd_file sub_cmd.cmd"
+  fi
+  ARGS="$ARGS -sdf_cmd_file sdf_cmd.cmd -sdf_verbose"
```

Comment:
Why is this here twice?

Reply:
Different files

Reply:
Got it. Didn't see the difference between sub_ and sdf_ at first.